### PR TITLE
Updated ci.rake usage of brakeman

### PR DESCRIPTION
**Pull request by justin808**
* Run brakeman like:
  `brakeman --exit-on-warn --quiet -A`
* This makes ci fail if there's a brakeman issue.
* Updated brakeman gem to current (3.1.1)
* Bonus: Added pry debugging gems

### CODE REVIEW

**Comment by justin808** [comment]
@dylangrafmyre @samnang Want to review?

File: lib/tasks/ci.rake [hunk 1]
```diff
@@ -8,7 +8,8 @@ if Rails.env.development?
 
   task :security_audit do
     puts Rainbow("Running security audit on code (brakeman)").green
-    Rake::Task["brakeman:run"].invoke("tmp/brakeman-report.html")
+
+    sh "brakeman --exit-on-warn --quiet -A -z"
```
Comment (dylangrafmyre) [inline lib/tasks/ci.rake]: `-z` AKA "hardcore mode" :)

**Comment by dylangrafmyre** [comment]
:+1: LGTM

**Comment by samnang** [comment]
:+1: LGTM

**Pull request merged**